third_party/ienn/src/ie_execution.cpp

```diff
@@ -158,16 +156,22 @@ int32_t Execution::StartCompute() {
     }
 
     infer_request->Infer();
-    OutputsDataMap outputInfo = compilation_->network_->getOutputsInfo();
-    auto outputInfoItem = outputInfo.begin();
+
     for (size_t i = 0; i < model->outputs.size(); ++i) {
       uint32_t index = model->outputs[i];
       const Operand& operand = model->operands[index];
       uint32_t offset = total_length;
       const uint32_t length = output_data_[i].length;
       total_length += length;
       void* mapping = output_data_[i].buffer;
-      Blob::Ptr output_blob = infer_request->GetBlob(outputInfoItem->first);
+      auto output_name =
+          compilation_->index_op_map_[index].get_node()->get_name();
+      // In case that one output layer has multiple outputs nodes
+      auto output_node_index = compilation_->index_op_map_[index].get_index();
+      if (output_node_index != 0) {
+        output_name.append(".").append(std::to_string(output_node_index));
+      }
+      Blob::Ptr output_blob = infer_request->GetBlob(output_name);
```

Comment:
Why input name does't need to append the index 0?
Is it used for one output node such as softmax or special topk?

Reply:
We use op::Parameter as input layer, this op only has one node with index 0. If index 0, the node name needn't append.